### PR TITLE
Upgrade grpc/protoc and all deps in the protobuf builder docker

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: protobuf/Dockerfile [hunk 1]
```diff
@@ -1,46 +1,59 @@
...
+        -GNinja \
+        -DBUILD_SHARED_LIBS=ON \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DgRPC_INSTALL=ON \
+        -DgRPC_BUILD_TESTS=OFF \
+        ../.. && \
+    cmake --build . --target plugins && \
+    cmake --build . --target install && \
+    DESTDIR=/out cmake --build . --target install
```
Comment (tigrannajaryan) [inline protobuf/Dockerfile:37]: Just curious: are these changes dictated by the new version of grpc? Did they move from make to cmake?

**Review by tigrannajaryan — APPROVED** [review]
I don't have the tools installed locally so don't have a way to test the change. I trust you verified it, so approving.

File: protobuf/Dockerfile (hunk 1 above)
Comment (bogdandrutu) [reply protobuf/Dockerfile:37]: Yes they now only use cmake and bazel for building.

**Comment by bogdandrutu** [comment]
@tigrannajaryan yes verified with the proto, also added support for php, ruby, cpp, csharp and everything. Soon you will see a PR there

**Comment by jpkrohling** [comment]
Did you rebase this based on the origin that we used for the Jaeger code?

**Comment by bogdandrutu** [comment]
@jpkrohling yes I used the original repo that you also used as inspiration

**Pull request merged**